packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -5,8 +5,14 @@ import {
   initializeWithHeadless,
```

Comment:
should we maybe add unit tests for the quanticSearchBox to cover whether to render the textarea or not? It currently has 1 test suite in the file which is pretty basic.

Reply:
Well it's tested with the Cypress tests for the standalone searchbox. We probably should be adding tests for the searchbox overall, but since we didn't have any I didn't take the time to add new ones to the searchbox alone, only on the standalone version which is exactly the same code.

I think in the future when we re-factor those two components so they actually are the same thing, we can then write tests to test both.